https://github.com/redhat-openstack/astapor/pull/95/files

diff -Naur -Naru astapor-openstack-foreman-installer-1.0.1.orig/puppet/modules/quickstack/manifests/controller_common.pp astapor-openstack-foreman-installer-1.0.1/puppet/modules/quickstack/manifests/controller_common.pp
--- astapor-openstack-foreman-installer-1.0.1.orig/puppet/modules/quickstack/manifests/controller_common.pp	2013-12-20 20:29:40.000000000 +0000
+++ astapor-openstack-foreman-installer-1.0.1/puppet/modules/quickstack/manifests/controller_common.pp	2014-01-07 12:06:19.505419346 +0000
@@ -183,8 +183,12 @@
     verbose                     => $verbose,
   }
 
-  package {'horizon-packages':
-    name   => ['python-memcached', 'python-netaddr'],
+  # horizon packages
+  package {'python-memcached':
+    ensure => installed,
+  }~>
+  package {'python-netaddr':
+    ensure => installed,
     notify => Class['horizon'],
   }
 
